src/main/java/com/ss/training/utopia/counter/entity/Flight.java

```diff
@@ -0,0 +1,121 @@
+package com.ss.training.utopia.counter.entity;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+
+/**
+ * @author Justin O'Brien
+ */
+@Table(name = "tbl_flight")
+@IdClass(FlightPk.class)
+public class Flight implements Serializable {
+
+	private static final long serialVersionUID = -8418526716923540697L;
+
+	@Id
+	@Column
+	private int departId, arriveId;
+
+	@Id
+	@Column
+	private LocalDateTime departTime;
+
+	@Column
+	private int flightId, seatsAvailable;
```

Comment:
I think we should make these 'int'  Integers instead for flexibility.

Reply:
That seems like a best practice. I seem to recall having to change an int to an Integer for some reason in some of the entity classes for the LMS. Also it gives me an excuse to overhaul all the data types for optimal memory use and so forth